package.json

```diff
@@ -47,6 +47,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
+    "ember-cli-update": "^0.27.0",
```

Comment:
We should probably omit this and use it globally, since we'll want to be using whatever the latest is, when we run it.